pyrenew/metaclass.py

```diff
@@ -125,6 +126,38 @@ def _assert_sample_and_rtype(
     return None
 
 
+def compute_delay_ascertained_incidence(
```

Comment:
Would suggest moving this out of `metaclass.py`. Why not put it in `convolve.py`

Reply:
Good point! Will move this function to `convolve.py`